CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CHANGELOG FOR `1.7.x`
 
+#### Details
+- [#217](https://github.com/Sylius/SyliusResourceBundle/issues/202) Duplicate initialisation of repositories ([@Fantus](https://github.com/Sylius/SyliusResourceBundle/issues/217))
```

Comment:
Thanks for adding it, but we are generating this notation based on commits. Note just for future reference